tests/Doctrine/Tests/DBAL/Functional/StatementTest.php

```diff
@@ -160,10 +160,10 @@ public function testReuseStatementWithParameterBoundByReference()
         $this->_conn->insert('stmt_test', array('id' => 1));
         $this->_conn->insert('stmt_test', array('id' => 2));
 
+        $id = 1;
         $stmt = $this->_conn->prepare('SELECT id FROM stmt_test WHERE id = ?');
         $stmt->bindParam(1, $id);
 
-        $id = 1;
```

Comment:
This is a false positive. `Statement::bindParam()` accepts the second argument by reference, so it's not supposed to be initialized by the moment when the method is called. As you can see, the test passes in its original state.